AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -474,14 +478,41 @@ private void RemoveSelectedObject(LayoutObject objectToRemove, bool includeSameO
             RemoveSelectedObjects(new List<LayoutObject>() { objectToRemove }, includeSameObjects);
         }
 
+        /// <summary>
+        /// Used to load current color for grid lines from settings.
+        /// </summary>
+        /// <remarks>Also calls <see cref="UIElement.InvalidateVisual()"/></remarks>
+        public void LoadGridLineColor()
+        {
+            var colorFromJson = SerializationHelper.LoadFromJsonString<SerializableColor>(_appSettings.ColorGridLines);//explicit variable to make debugging easier
+            _gridLinePen = _penCache.GetPen(_brushCache.GetSolidBrush(colorFromJson), DPI_FACTOR * 1);
+
+            InvalidateVisual();
+        }
+
+        /// <summary>
+        /// Used to load current color for object border lines from settings.
+        /// </summary>
+        /// <remarks>Also calls <see cref="UIElement.InvalidateVisual()"/></remarks>
+        public void LoadObjectBorderLineColor()
+        {
+            var colorFromJson = SerializationHelper.LoadFromJsonString<SerializableColor>(_appSettings.ColorObjectBorderLines);//explicit variable to make debugging easier
+            _linePen = _penCache.GetPen(_brushCache.GetSolidBrush(colorFromJson), DPI_FACTOR * 1);
+        }
```

Comment:
> Also calls <see cref="UIElement.InvalidateVisual()"/>
Doesn't appear to call `InvalidateVisual()` - maybe missing a call here?

Reply:
I hate copy/paste ...